retriever/lib/install.py

```diff
@@ -1,209 +0,0 @@
-from __future__ import print_function
-from __future__ import absolute_import
-import imp
-import os
-import sys
-from pkg_resources import parse_version
-from retriever.engines import choose_engine
-from retriever.lib.compile import compile_json
-from retriever.lib.defaults import DATA_DIR, SCRIPT_SEARCH_PATHS, VERSION
-from retriever.lib.tools import name_matches
-from retriever.lib.scripts import SCRIPT_LIST
-
-script_list = SCRIPT_LIST()
```

Comment:
Yes, that is right. When there are no available scripts the `install_json` function or any other install function won't work. The user will have to first update the scripts and then run the install functions:
```python
import retriever as rt
rt.check_for_updates()
rt.install_json("iris")
```
I am not sure if this is what you were trying to say.